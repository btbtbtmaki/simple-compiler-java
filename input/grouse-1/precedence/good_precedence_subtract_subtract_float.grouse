main {

  print 6.4 - 9.2 - 1.0 nl;
  print 5.6 - 8.4 - 9.5 nl;
  print 7.9 - 9.8 - 9.8 nl;
  print 9.0 - 1.8 - 9.2 nl;
  print 4.0 - 7.7 - 2.9 nl;
  print 4.2 - 9.3 - 8.8 nl;
  print 2.2 - 3.8 - 7.0 nl;
  print 3.3 - 4.1 - 3.6 nl;
  print 5.4 - 3.5 - 4.5 nl;
  print 6.6 - 2.0 - 3.8 nl;
  print 8.3 - 7.3 - 2.6 nl;
  print 7.3 - 6.6 - 4.5 nl;

  print 8.2 - 2.1 - 6.3 nl;
  print 7.6 - 5.1 - 3.9 nl;
  print 6.5 - 3.9 - 9.4 nl;
  print 4.4 - 2.3 - 6.9 nl;
  print 2.7 - 1.7 - 3.1 nl;
  print 6.2 - 5.0 - 5.2 nl;
  print 1.2 - 9.4 - 7.3 nl;
  print 3.1 - 7.2 - 7.0 nl;
  print 5.8 - 3.1 - 3.1 nl;
  print 8.6 - 1.4 - 4.0 nl;
  print 4.5 - 3.5 - 4.0 nl;
  print 9.3 - 4.3 - 6.0 nl;

  print 5.2 - 3.7 - 3.5 - 3.3 nl;
  print 1.9 - 1.2 - 5.6 - 7.6 nl;
  print 4.1 - 6.1 - 8.5 - 1.6 nl;
  print 9.9 - 3.8 - 2.3 - 9.7 nl;
  print 9.0 - 3.2 - 9.0 - 8.2 nl;
  print 4.7 - 1.4 - 5.7 - 6.0 nl;
  print 5.7 - 5.1 - 2.2 - 6.7 nl;
  print 8.7 - 9.4 - 1.7 - 9.3 nl;
  print 3.6 - 3.8 - 9.4 - 8.3 nl;
  print 7.0 - 9.0 - 4.5 - 5.6 nl;
  print 9.2 - 1.4 - 1.1 - 4.0 nl;
  print 1.6 - 9.2 - 9.7 - 5.3 nl;

  print 7.7 - 9.7 - 7.6 - 7.0 nl;
  print 6.2 - 3.7 - 8.3 - 6.0 nl;
  print 8.0 - 4.6 - 7.8 - 7.6 nl;
  print 3.9 - 2.7 - 3.7 - 7.6 nl;
  print 9.9 - 9.0 - 7.2 - 6.8 nl;
  print 4.6 - 8.5 - 4.5 - 4.8 nl;
  print 7.2 - 6.4 - 6.8 - 1.5 nl;
  print 4.4 - 9.8 - 3.6 - 8.2 nl;
  print 6.9 - 9.4 - 5.8 - 6.5 nl;
  print 9.5 - 2.7 - 3.9 - 2.4 nl;
  print 9.0 - 8.8 - 9.6 - 4.9 nl;
  print 5.3 - 7.4 - 5.3 - 4.9 nl;

}

