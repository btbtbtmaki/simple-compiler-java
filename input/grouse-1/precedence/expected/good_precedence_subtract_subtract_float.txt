-3.8
-12.3
-11.7
-2
-6.6
-13.9
-8.6
-4.4
-2.6
0.8
-1.6
-3.8
-0.2
-1.4
-6.8
-4.8
-2.1
-4
-15.5
-11.1
-0.4
3.2
-3
-1
-5.3
-12.5
-12.1
-5.9
-11.4
-8.4
-8.3
-11.7
-17.9
-12.1
2.7
-22.6
-16.6
-11.8
-12
-10.1
-13.1
-13.2
-7.5
-17.2
-14.8
0.5
-14.3
-12.3
